.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```

Comment:
what is it for?

Reply:
This got pushed here accidentally. It has nothing to do with the program.